workers/scanner.go

```diff
@@ -327,7 +328,12 @@ func (s *Scanner) managedFindAndPinOneUnderpinnedSkylink() (skylink skymodules.S
 	minPinners := s.minPinners
 	s.mu.Unlock()
 
-	sl, err := s.staticDB.FindAndLockUnderpinned(context.TODO(), s.staticServerName, minPinners)
+	ctx := context.TODO()
+	// skipSkylinks is a list of skylinks which we want to skip during this
+	// scan. These might be skylinks which errored out or blocked skylinks.
+	skipSkylinks := make([]string, 0)
```

Comment:
Could be `var skipSkylinks []string` I think. For loops and `len` should still handle it correctly.

Reply:
Sure. I wanted to be sure that `FindAndLockUnderpinned` will be fine using it because I don't really trust the mongo driver to handle nil slices correctly. But looking at this again, I should make handle that in `FindAndLockUnderpinned` instead.

Changed.